turbinia/output_manager.py

```diff
@@ -383,13 +391,24 @@ def create_output_dir(self, base_path=None):
     pass
 
   def copy_to(self, source_path):
+    if os.path.getsize(source_path) == 0:
+      msg = 'Local source file {0:s} is empty.  Not uploading to GCS'.format(
+          source_path)
+      log.error(msg)
```

Comment:
Per the style guide, this should be "message", not "msg"

Reply:
Thanks for the review.  I changed all instances of message/exception in these files.  I should probably take a wider pass, but I'll do that in a separate PR since there are lots of places where that shows up.